res/css/photon/message-bar.css

```diff
@@ -5,53 +5,68 @@
 /* See https://design.firefox.com/photon/components/message-bars.html */
 .photon-message-bar {
   display: flex;
-  width: 100%;
   min-height: 32px;
   box-sizing: border-box;
   align-items: center;
-  padding: 0 4px;
+
+  /* Note: 32px is: 4px (left padding for the message bar) + 4px (left padding
+   * for the icon) + 16px (icon width) + 4px (right padding for the icon) + 4px
+   * (space between icon and text) */
+  padding: 4px 4px 4px 32px;
+
+  /* Note: 8px is: 4px (left padding for the message bar) + 4px (left padding
+   * for the icon). And same for the top positioning, because we want the icon
+   * to stick at the top when the text is multiline */
+  background: url(../../img/svg/info-icon.svg) no-repeat 8px 8px / 16px 16px
+    var(--grey-20);
   border-radius: 4px;
+  color: var(--grey-90);
   font-size: 13px;
   font-weight: 400;
   line-height: 1.4;
 }
 
-.photon-message-bar .photon-button {
-  margin-left: 8px;
+.photon-message-bar-action-button {
+  flex: none;
+  margin: 0 8px;
 }
 
-.photon-message-bar .photon-message-bar-close-button {
+.photon-message-bar-close-button {
```

Comment:
I simplified this selector, it's more targeted now. It works fine because the specificity wasn't useful either.